package.json

```diff
@@ -85,8 +85,8 @@
     "babel-cli": "^6.24.0",
     "babel-eslint": "^7.2.1",
     "babel-plugin-transform-runtime": "^6.23.0",
-    "chrome-launcher": "^0.1.1",
-    "chrome-remote-interface": "^0.23.2",
+    "chrome-launcher": "^0.10.2",
+    "chrome-remote-interface": "^0.26.1",
```

Comment:
We have to make this `0.25.x` because of https://github.com/cyrus-and/chrome-remote-interface/commit/b4f8715dbe93185d7f2102ddbcfebb04f804fa04

Reply:
@dandv Any updates on this? :)

Reply:
Feel free to patch my PR and run with it!